src/Infrastructure/Service/LifecycleManager.php

```diff
@@ -51,4 +78,79 @@ public function update(): void
             $action->apply($taskIds, $folderTasks, $databaseTasks);
         }
     }
+
+    /**
+     * @throws \SprykerSdk\Sdk\Infrastructure\Exception\SdkVersionNotFoundException
+     *
+     * @return array<\SprykerSdk\SdkContracts\Entity\MessageInterface>
+     */
+    public function checkForUpdate(): array
+    {
+        $versionFilePath = $this->sdkDirectory . '/VERSION';
+
+        if (!file_exists($versionFilePath)) {
+            throw new SdkVersionNotFoundException('Could not find VERSION file, skip updatable check');
+        }
+
+        $currentVersion = file_get_contents($versionFilePath);
+
+        if (!$currentVersion) {
+            throw new SdkVersionNotFoundException('Could not find VERSION file, skip updatable check');
```

Comment:
Update exception message related to empty file

Reply:
Fixed